docs/section11/SHW comparison fields.yaml

```diff
@@ -0,0 +1,396 @@
+ServiceWaterHeatingDistributionSystem:
+  Data Elements:
+    id:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    reporting_name:
+      AppG 11-1 P_RMD Equals U_RMD: false
+      AppG 11-3 P_RMD Equals B_RMD: false
+    notes:
+      AppG 11-1 P_RMD Equals U_RMD: false
+      AppG 11-3 P_RMD Equals B_RMD: false
+    design_supply_temperature:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    design_supply_temperature_difference:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    tanks:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    is_central_system:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    service_water_piping:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    distribution_compactness:
+      AppG 11-1 P_RMD Equals U_RMD: false
+      AppG 11-3 P_RMD Equals B_RMD: false
+    control_type:
+      AppG 11-1 P_RMD Equals U_RMD: false
+      AppG 11-3 P_RMD Equals B_RMD: false
+    configuration_type:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    is_recovered_heat_from_drain_used_by_water_heater:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: false
+    drain_heat_recovery_efficiency:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: false
+    drain_heat_recovery_type:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: false
+    flow_multiplier_schedule:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: false
```

Comment:
I am not remembering why we thougth this could change. If it is a tenant fitout then a SWH system would exist and that would be modeled so I can't think of a scenario where this would be false. Can you please remind me?

Reply:
@claperle and @KarenWGard could this be the case when there are low flow fixtures in the P RMD?

Reply:
@supriyagoel I think that is the reason I think Karen chose fail here but I was trying to remember when there could be low flow fixtures in the proposed but no (1) existing SWH or (2)  no water heater as part of the design. I was thinking if it was a tenent fit-out there would either be an existing water heater or a water heater as part of the design and if it was core and shell then the low flow fixtures were not be relevant. I guess maybe there is a a rare scenario where someone could be doing an interior fitout where the central SWH systems are yet to be designed?

Reply:
change to true - we expect the flow rate to be adjusted by the design_flow_rate, not by the schedule